infra/_modules/cms_function_app/networking.tf

```diff
@@ -0,0 +1,20 @@
+resource "azurerm_private_endpoint" "st_queuest_queue" {
+  name                = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}-queuest-queue-pep-01"
+  location            = var.location
+  resource_group_name = var.resource_group_name
+  subnet_id           = var.peps_snet_id
+
+  private_service_connection {
+    name                           = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}-queuest-queue-pep-01"
```

Comment:
**chore:** use `cms` instead of `queue` to indicate this private service connection is related to `cms` storage account
```suggestion
    name                           = "${var.prefix}-${var.env_short}-${var.location_short}-${var.domain}-cms-st-queue-pep-01"
```